internal/reconcilers/artifacts.go

```diff
@@ -154,93 +150,19 @@ func (e *Reconciler) handleArtifactsReconcilerEvent(ctx context.Context, evt *Re
 		return err
 	}
 
-	// create artifact verifier
-	artifactVerifier, err := verifier.NewVerifier(
-		verifier.VerifierSigstore,
-		sigstore.SigstorePublicTrustedRootRepo,
-		container.WithAccessToken(cli.GetToken()), container.WithGitHubClient(cli))
-	if err != nil {
-		return fmt.Errorf("error getting sigstore verifier: %w", err)
-	}
-	defer artifactVerifier.ClearCache()
-
 	for _, artifact := range artifacts {
 		// store information if we do not have it
+		typeLower := strings.ToLower(artifact.GetPackageType())
 		newArtifact, err := e.store.UpsertArtifact(ctx,
 			db.UpsertArtifactParams{RepositoryID: repository.ID, ArtifactName: artifact.GetName(),
-				ArtifactType: artifact.GetPackageType(), ArtifactVisibility: artifact.GetVisibility()})
+				ArtifactType: typeLower, ArtifactVisibility: artifact.GetVisibility()})
```

Comment:
Will switching this leave any orphaned rows in the database?

Reply:
A: we're orphaning _all_ the rows in the database.  But that's on purpose, and helps make this less stateful and more aligned with the other verifiers.

Reply:
wait, I think your comment was on point. There are two tables - ArtifactVersions that we are orphaning completely and Artifacts that we are keeping. This changes the way we store Artifacts and you are right we would have orphaned rows and that we need to handle this change one way or the other (IOW, I treat this comment as Changes Requested).

me and @rdimitrov just ran into another issue on Friday related to how we verify artifacts signed by a private sigstore instance that cost some time.

Reply:
Actually sorry, I should have read the DB code better before replying. I think we can't leave any orphaned rows in the database[*] because we already have `ON CONFLICT` for a tuple consisting of `(repo_id, name)` which means that on update with a differently-cased type, we'd just have switched the type:
```
INSERT INTO artifacts (
    repository_id,
    artifact_name,
    artifact_type,
    artifact_visibility
) VALUES ($1, $2, $3, $4)
ON CONFLICT (repository_id, LOWER(artifact_name))
DO UPDATE SET
    artifact_type = $3,
    artifact_visibility = $4
WHERE artifacts.repository_id = $1 AND artifacts.artifact_name = $2
RETURNING *;
```

[*] This is also not ideal because it means that we can't support artifacts that have the same name but a different type, e.g. a ruby gem called `foo` and a container called `foo` but for this specific case, the DB API bug actually helps us.